packages/client/mutations/CreateTaskIntegrationMutation.ts

```diff
@@ -0,0 +1,184 @@
+import graphql from 'babel-plugin-relay/macro'
+import {stateToHTML} from 'draft-js-export-html'
+import {commitMutation} from 'react-relay'
+import {StandardMutation} from '../types/relayMutations'
+import splitDraftContent from '../utils/draftjs/splitDraftContent'
+import makeSuggestedIntegrationId from '../utils/makeSuggestedIntegrationId'
+import createProxyRecord from '../utils/relay/createProxyRecord'
+import {CreateTaskIntegrationMutation as TCreateTaskIntegrationMutation} from '../__generated__/CreateTaskIntegrationMutation.graphql'
+
+graphql`
+  fragment CreateTaskIntegrationMutation_task on CreateTaskIntegrationPayload {
+    task {
+      ...IntegratedTaskContent_task
+      integration {
+        __typename
+        ... on JiraIssue {
+          cloudId
+          cloudName
+          url
+          issueKey
+          summary
+          descriptionHTML
+          projectKey
+          project {
+            name
+          }
+        }
+        ... on _xGitHubIssue {
+          bodyHTML
+          title
+          number
+          repository {
+            nameWithOwner
+          }
+        }
+        ...TaskIntegrationLinkIntegrationGitHub
+        ...TaskIntegrationLinkIntegrationJira @relay(mask: false)
```

Comment:
Found the bug! It's in the code that I wrote for nest-graphql-endpoint.

That package must be removing this inline fragment but still sending the fragment body.
I'll push a fix for it 

-1 this is stuck on me pushing a fix for nest-graphql-endpoint.

Reply:
fix pushed directly to the PR (it's just a change to package.json).

If you're curious, the code to fix it is here:  https://github.com/ParabolInc/nest-graphql-endpoint/commit/bc4b6fb482f9cb5180deae525d2a4f27e035e9a9#diff-56b72bf7eed61753d4b74cd03f03f5a6d152685f1ca5f055db96ac1c58fb2de6

loom: https://www.loom.com/share/e9c3e95cae6248e6bd9c139e945fa84b